pilot/prompts/dev_ops/ran_command.prompt

```diff
@@ -1,7 +1,7 @@
-{{ additional_info }}I ran the command `{{ command }}`
-{%- if exit_code is number %}, the exit code was {{ exit_code }}{% endif %}
-{%- if error_response %}, it {{ error_response }}{% endif %}. The output was:
-
+{%- if additional_message %}{{ additional_message }}{% endif %}
+I ran the command `{{ command }}`
+{%- if exit_code is number %}, the exit code was {{ exit_code }}{% endif %}. The output was:
+{#%- if error_response %}, it {{ error_response }}{% endif %#}
```

Comment:
what's `it` here? Maybe we could do something like:

```
The output was:
{{ cli_response }}
{% if error_response %}The standard error output was:
{{ error_response }}
{% endif %}

(maybe that's what you meant)

Reply:
With previous implementation error_response could be:

- 'was interrupted by user' then message in prompt would be: ', it was interrupted by user. The output was:'
- 'took longer than {timeout}ms so I killed it' then message in prompt would be: ', it took longer than {timeout}ms so I killed it. The output was:'

now I commented out error_response in prompt.